src/App/components/Preview.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import './Preview.scss';
 
 const HIGHLIGHT_VERSE_INDEX = 0;
-const HIGHLIGHT_WORD_INDEX = 3;
+const HIGHLIGHT_WORD_INDEX = [0,1,2];
```

Comment:
It would be better to rename this to `HIGHLIGHT_WORD_INDEXES` since it is now an array of indexes (plural) rather then a single index.

Reply:
Good call